CHANGELOG/1.3-CHANGELOG.md

```diff
@@ -0,0 +1,232 @@
+## Changes by Kind
+
+### Noteworthy Changes for Implementors
+This section is intended to be a guide for API changes that might inspire or require implementation changes.
+None of these API changes represent breaking changes (?).
+
+#### OverlappingTLSConfig for Connection Coalescing
+A new `OverlappingTLSConfig` condition has been added to Gateway Listeners to indicate situations where
+Connection Coalescing could be problematic.  The Gateway specification for handling Hostname and SNI matching for HTTPS
+requests has been clarified and now recommends that implementations return 421 HTTP code responses in certain cases.
+
+- Implementation of GEP-3567 - TLS Updates for Connection Coalescing. ([#3630](https://github.com/kubernetes-sigs/gateway-api/pull/3630), [@robscott](https://github.com/robscott))
+- Add GEP-3567: Gateway TLS Updates for HTTP/2 Connection Coalescing. ([#3572](https://github.com/kubernetes-sigs/gateway-api/pull/3630), [@robscott](https://github.com/robscott))
+
+#### Move `BackendTLSPolicy` `SubjectAltNames` from Core to Extended
+- The `SubjectAltNames` field of `BackendTLSPolicy` changed from Core to Extended feature. ([#3591](https://github.com/kubernetes-sigs/gateway-api/pull/3591), [@mlavacca](https://github.com/mlavacca))
+
+#### `backendRef` filter must send traffic to the correct backends when weighted routing is configured
+- A new conformance test was added to ensure `backendRef` filters don't affect weighted routing. ([#3604](https://github.com/kubernetes-sigs/gateway-api/pull/3604),[@dprotaso](https://github.com/dprotaso))
+
+#### Clarify reasons for certain object status conditions
+- Set proper reason for Gateway `parametersRef` `Accepted` condition when `parametersRef` is invalid. ([#3579](),[@mlavacca]())
+- Improve GatewayClass `GatewayClassReasonInvalidParameters` reason description. ([#3553](https://github.com/kubernetes-sigs/gateway-api/pull/3553)[@mlavacca](https://github.com/mlavacca))
+
+### BackendTLSPolicy
+- CEL validation for target references in `BackendTLSPolicy`. ([#3496](https://github.com/kubernetes-sigs/gateway-api/pull/3496), [@snorwin](https://github.com/snorwin))
+ 
+## Standard Channel Additions and Changes
+The Standard channel is Gateway API's set of maximally-stable install files.
+Only features with the best testing and support are added to the standard
+channel. This channel should be considered GA or stable, and future changes will
+be fully backwards compatible.
+
+### Percentage-Based Request Mirroring 🎉 (This section needs thorough review)
+This feature enhances the existing [request mirroring feature](https://gateway-api.sigs.k8s.io/guides/http-request-mirroring/)
+by allowing users to specify a percentage of requests to be mirrored in both `HTTPRoute` (and `GRPCRoute`?) objects.
+
+This feature has graduated to Standard and is now considered GA or Stable.
+
+This feature's name for conformance tests (and GatewayClass status reporting?) is
+`HTTPRouteRequestPercentageMirror`(?).
```

Comment:
Yes, this is correct - the definition is in `conformance/pkg/features/httproute.go`. (I had to hunt around to find the correct file too.)

Reply:
I note that the `httproute.go` does not mark this as Standard, however. I'll do a check of all of the listed features and do an update PR in a moment.

Reply:
#3696 does that change.

Reply:
Thanks, I'll add some more commits to the list.